modules/nf-core/annotsv/Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM mambaorg/micromamba:1.4-focal
+
+LABEL authors="Nicolas Vannieuwkerke <nicolas.vannieuwkerke@ugent.be>" \
+    description="Docker image containing AnnotSV"
+
+ENV VERSION=3.3.2
+ENV ANNOTSV=/tmp/AnnotSV-${VERSION}
```

Comment:
I've noticed 😁. Thanks for the suggestion I was failing miserably in fixing this.

Reply:
Just to note; I don't know how their install looks, might be required to do a `mkdir -p "$ANNOTSV"` before installing.

Reply:
Would you know why when converting to a singularity image, the files are deleted from the directory they were downloaded in (It's in `/home/mambauser` in the current version, because of permission issues with `/opt`). The tool needs to have access to the file structure like when it's downloaded to fully work. (For now when converting to singularity, the AnnotSV executable gets moved to `/usr/local/bin` instead)

Reply:
> Just to note; I don't know how their install looks, might be required to do a `mkdir -p "$ANNOTSV"` before installing.

That variable should point to the directory containing the source code of AnnotSV

Reply:
> > Just to note; I don't know how their install looks, might be required to do a `mkdir -p "$ANNOTSV"` before installing.
> 
> That variable should point to the directory containing the source code of AnnotSV

It doesn't look like it's used like that - instead the source is fetched and extracted to / - instead, the variable is used as installation prefix.

Reply:
Yes indeed, but the main problem is that it's also used to find to location of a dependency which is in the source directory. It can't find this dependency if this variable points to another location :/



> Anything in /home is also bad, either switch back and forth with `USER` or make it multi-stage and copy over between?

What do you mean by multi-stage exactly?

Reply:
> What do you mean by multi-stage exactly?

Multi-stage are builds that are done in multiple stages, to allow for using less layers in the final image/not including tooling needed for building.

Are you sure it's really expecting to look at the source directory?

https://github.com/pontus/nf-core-modules/tree/new-module-annotsv/modules/nf-core/annotsv is an example of how I think it should be after a quick look at AnnotSV (I'm not sure about licensing of the annotations or phenotypes fetched by make install and thus that may need to be addressed).

Reply:
Thanks for the explanation it's clear to me now! I've been working with the developer to make the tool more compatible (and eventually add it to bioconda). We've made some major improvements which will make sure we won't have to do this 😄